docs/book/user-guide/production-guide/configure-pipeline.md

```diff
@@ -1,17 +1,132 @@
+
+{% hint style="info" %}
+The `with_options` command that points to a YAML config is only one way to configure a pipeline. We can also directly configure a pipeline or a step in the decorator:
+
+```python
+@pipeline(settings=...)
+```
+
+However, it is best to not mix configuration from code to ensure seperation of concerns in our codebase.
 {% endhint %}
 
-This section of the production guide will be about best practices on how to configure a pipeline in production. Until this section is ready, here are some useful links to achieve this goal:
+## Breaking down our configuration YAML
+
+The YAML configuration of a ZenML pipeline can be very simple, as in this case. Let's break it down and go through each section one by one:
+
+### The docker settings
+
+```yaml
+settings:
+  docker:
+    required_integrations:
+      - sklearn
+    requirements:
+      - pyarrow
+```
+
+The first section is the so called `settings` of the pipeline. This section has a `docker` key, which controls the [containerization process](cloud-orchestration.md#orchestrating-pipelines-on-the-cloud). Here, we are simply telling ZenML that we need `pyarrow` as a pip requirement, and we want to enable to `sklearn` integration of ZenML, which will in turn install the `scikit-learn` library. This docker section can be populated with many different options, and correspond to the [DockerSettings](https://sdkdocs.zenml.io/latest/core_code_docs/core-config/#zenml.config.docker_settings.DockerSettings) class in the Python SDK.
+
+### Associating a ZenML Model
+
+The next section is about associating a [ZenML Model](../starter-guide/track-ml-models.md) with this pipeline.
+
+```yaml
+# Configuration of the Model Control Plane
+model_version:
+  name: breast_cancer_classifier
+  version: rf
+  license: Apache 2.0
+  description: A breast cancer classifier
+  tags: ["breast_cancer", "classifier"]
+```
+
+You will see that this configuration lines up with the model created after executing these pipelines:
+
+{% tabs %}
+{% tab title="CLI" %}
+```shell
+# List all versions of the breast_cancer_classifier
+zenml model version list breast_cancer_classifier
+```
+{% endtab %}
+{% tab title="Dashboard" %}
+The ZenML Cloud ships with a Model Control Plane dashboard where you can visualize all the versions:
+
+<figure><img src="../../.gitbook/assets/mcp_model_versions_list.png" alt=""><figcaption><p>All model versions listed</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+### Passing parameters
+
+```yaml
+# Configure the pipeline
+parameters:
+  model_type: "rf"  # Choose between rf/sgd
+```
+
+The last part of the config YAML is the `parameters` key. This parameters key aligns with the parameters that the pipeline expects. In this case, the pipeline expects a string called `model_type` that will inform it which type of model to use:
```

Comment:
```suggestion
### Passing parameters

The last part of the config YAML is the `parameters` key.

```yaml
# Configure the pipeline
parameters:
  model_type: "rf"  # Choose between rf/sgd
```
This parameters key aligns with the parameters that the pipeline expects. In this case, the pipeline expects a string called `model_type` that will inform it which type of model to use:
```

Reply:
Can't leave a properly formatted change request here for some reason. I would move the sentence ```The last part of the config YAML is the `parameters` key.``` above the code block here.